Dockerfile.travis

```diff
@@ -28,9 +28,10 @@ WORKDIR /src/healthcare-application
 # Ensure any deltas in package.json are reflected.
 COPY package.json package.json
 COPY npm-shrinkwrap.json npm-shrinkwrap.json
+COPY webpack.config.js webpack.config.js
 
 ENV NODE_ENV development
 RUN npm install
-
+RUN npm run webpack-prod
```

Comment:
Move this to after line 37 `ADD . .`

Then delete line 32 `COPY webpack.config.js webpack.config.js`

And it should all still work.